.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,7 +1,6 @@
 build.commands=org.eclipse.jdt.core.javabuilder
 connection.arguments=
 connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
-connection.gradle.user.home=null
```

Comment:
Er the removal of this line is causing issues with our Gradle build...

Could you check whether your Eclipse is unintentionally removing this line?